core/scripts/sql/texera_ddl.sql

```diff
@@ -20,6 +20,15 @@ CREATE TABLE IF NOT EXISTS user
 -- start auto increment userID from 1 because userID 0 means user not exists
   AUTO_INCREMENT = 1;
 
+
+CREATE TABLE IF NOT EXISTS google_user
+(
+    `uid` VARCHAR(256)                 NOT NULL,
+    `name` VARCHAR(32)                 NOT NULL,
+    PRIMARY KEY (`uid`)
+) ENGINE = INNODB;
+
+
```

Comment:
please make sure the format is aligned with other blocks. you can ignore it if it is already aligned.